src/main/java/frc/robot/Constants.java

```diff
@@ -74,11 +76,21 @@ public static class constDrivetrain {
      */
     public static final LinearVelocity REAL_DRIVE_SPEED = Units.FeetPerSecond.of(15.1);
     // Physically measured from center to center of the wheels
+    // Distance between Left & Right Wheels for 25 by 25 frame
+    // TODO: Replace with a measurement of the actual robot
+    public static final double TRACK_WIDTH_25 = Units.Meters.convertFrom(0, Units.Inches);
+    // Distance between Front & Back Wheels for 25 by 25 frame
+    public static final double WHEELBASE_25 = Units.Meters.convertFrom(0, Units.Inches);
```

Comment:
Use the 2024 Offbot code to get these numbers (then you can remove the TODO comment)